onnxruntime/core/providers/iree/compiler/torch-mlir-import-onnx/OnnxImporter.cpp

```diff
@@ -387,6 +387,14 @@ ContextCache::ConvertTensorProtoToAttr(const onnx::TensorProto &tp) {
     case onnx::TensorProto::DataType::TensorProto_DataType_FLOAT:
       return mlirDenseElementsAttrFloatGet(tensor_type, tp.float_data_size(),
                                            tp.float_data().data());
+    case onnx::TensorProto::DataType::TensorProto_DataType_INT8: {
+      std::vector<int8_t> int8_conversion;
+      int8_conversion.reserve(tp.int32_data_size());
```

Comment:
Is the tensor proto data stored as int32's even if the dtype is int8? I'm not sure I understand this.

Reply:
Ah, looking at <https://github.com/microsoft/onnxruntime/blob/ec63d10303d52f1bd9d1b56823da6241365f817b/onnxruntime/core/framework/tensorprotoutils.cc#L264> I'm starting to get it. Weird.